### PR TITLE
RT-307: Feature/load test

### PR DESCRIPTION
**Pull request by dev-digital**
**Changes**
- Adding locust related packages to app's env
- Create load test tool config and scenario file
- Update README

### CODE REVIEW

**Review by NickFitz — APPROVED** [review]
:+1

File: requirements.txt [hunk 1]
```diff
@@ -67,8 +80,9 @@ text-unidecode==1.3
 toml==0.10.2
 typed-ast==1.4.2
 typing-extensions==3.7.4.3
-urllib3==1.26.5
+urllib3==1.26.4
```
Comment (mforner13) [inline requirements.txt:83]: Has reverted urllib back to previous version

File: README.md [hunk 2]
```diff
@@ -34,13 +34,31 @@ Where it is not possible to use a govuk style, the [moj-frontend](https://github
...
+### To start load test
+
+- Start *dev* or *testserver* in one terminal
+- Start locust tool in another terminal with below command
+
+```python
+locust --config .locust.conf
+```
+
+Refer tool's docs for more info on the flags used.
```
Comment (mforner13) [inline README.md:60]: 'Refer to the tool's docs' would be clearer :)

File: requirements.txt (hunk 1 above)
Comment (mforner13) [inline requirements.txt:83]: Has reverted urllib back to a previous version

File: README.md (hunk 2 above)
Comment (mforner13) [inline README.md:60]: Refer to the tool's docs would be clearer :)

**Review by mforner13 — COMMENTED** [review]
Two v minor changes :)

File: requirements.txt (hunk 1 above)
Comment (dev-digital) [reply requirements.txt]: Reverted it. Not sure why pip did that.

File: README.md (hunk 2 above)
Comment (dev-digital) [reply README.md]: Done, surprisingly google didn't pick this up.

**Review by mforner13 — APPROVED** [review]
👍  nice one!

**Pull request merged**